tidegates/tests/test_utils.py

```diff
@@ -399,8 +399,9 @@ class _polygons_to_raster_mixin(object):
     testfile = resource_filename("tidegates.testing.polygons_to_raster", "test_zones.shp")
     known_values = numpy.array([-999, 16, 150])
 
+    @nptest.dec.skipif(not tgtest.has_spatial)
     def test_process(self):
-        raster = utils.polygons_to_raster(self.testfile, "GeoID", **self.kwargs)
+        raster = utils.polygons_to_raster(self.testfile, "GeoID")
```

Comment:
Right, instead the function defaults to a cellsize of 4. I'll see if there's another way to get around the error it was throwing.